app/views/state_file/questions/md_tax_refund/edit.html.erb

```diff
@@ -18,7 +18,7 @@
                 :payment_or_deposit_type,
                 collection: [
                   { value: :direct_deposit, label: t("state_file.questions.tax_refund.edit.direct_deposit"), input_html: { "data-follow-up": "#deposit-to-bank" } },
-                  { value: :mail, label: t("state_file.questions.tax_refund.edit.mail"), input_html: { "data-follow-up": "#mail-check" } },
```

Comment:
hmm, why'd you remove this?

Reply:
we are not using it. There is only a reveal for direct deposit.